src/server/src/users/entities/user.entity.ts

```diff
@@ -38,6 +38,20 @@ export class User implements IUser {
   @Exclude()
   passwordHash: string;
 
+  @Column({
+    type: "timestamp with time zone",
+    name: "user_created_dt",
```

Comment:
I'm curious - is there a reason you went with `user_created_dt` instead of just `created_dt`, but named the field `createdDt` instead of `userCreatedDt`?
```suggestion
    name: "created_dt",
```

Reply:
Ah, I thought since there were other tables with "created_dt"-named columns I should distinguish the column name to make it easier in the event of a join but now realizing that's mostly just confusing/irrelevant.